### PR TITLE
Add I_S.keywords for v7.5

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

Add I_S.keywords for v7.5
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/18155
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: develop/dev-guide-choose-driver-or-orm.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ TiDB 兼容 MySQL 的协议，但存在部分与 MySQL 不兼容或有差异的
 > **注意：**
 >
 > - 在 8.0.32 之前的 MySQL Connector/J 8.0 版本中存在一个 [bug](https://bugs.mysql.com/bug.php?id=106252)，当与 TiDB v6.3.0 之前的版本一起使用时，可能会导致线程卡死。为了避免此问题，建议使用 MySQL Connector/J 8.0.32 或更高版本，或者使用 TiDB JDBC（见 *TiDB-JDBC* 标签）。
-> - MySQL Connector/J 8.0 与 TiDB v7.5.0 一起使用时，建议将 TiDB 配置项 [`server-version`](/tidb-configuration-file.md#server-version) 设置为 `"5.7.25-TiDB-v7.5.0"`。这是因为，当 TiDB 服务器报告版本为 MySQL 8.0.11 或更高版本时，MySQL Connector/J 会尝试访问 TiDB v7.5.0 中不存在的 `information_schema.KEYWORDS` 表。
+> - MySQL Connector/J 8.0 与 TiDB v7.5.2 或之前版本一起使用时，建议将 TiDB 配置项 [`server-version`](/tidb-configuration-file.md#server-version) 设置为 `"5.7.25-TiDB-v7.5.x"`。这是因为，当 TiDB 服务器报告版本为 MySQL 8.0.11 或更高版本时，MySQL Connector/J 会尝试访问 TiDB v7.5.2 及之前版本中不存在的 `information_schema.KEYWORDS` 表。
```
Comment (Oreoxmt) [inline develop/dev-guide-choose-driver-or-orm.md:35]:
```suggestion
> - MySQL Connector/J 8.0 与 TiDB v7.5.2 或之前版本一起使用时，建议将 TiDB 配置项 [`server-version`](/tidb-configuration-file.md#server-version) 设置为 `"5.7.25-TiDB-v7.5.x"`。这是因为，当 TiDB 服务器报告版本为 MySQL 8.0.11 或更高版本时，MySQL Connector/J 会尝试访问 TiDB v7.5.2 及之前版本中不存在的 [`information_schema.KEYWORDS`](/information-schema/information-schema-keywords.md) 表。
```

File: develop/dev-guide-choose-driver-or-orm.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ TiDB 兼容 MySQL 的协议，但存在部分与 MySQL 不兼容或有差异的
 > **注意：**
 >
 > - 在 8.0.32 之前的 MySQL Connector/J 8.0 版本中存在一个 [bug](https://bugs.mysql.com/bug.php?id=106252)，当与 TiDB v6.3.0 之前的版本一起使用时，可能会导致线程卡死。为了避免此问题，建议使用 MySQL Connector/J 8.0.32 或更高版本，或者使用 TiDB JDBC（见 *TiDB-JDBC* 标签）。
-> - MySQL Connector/J 8.0 与 TiDB v7.5.x 一起使用时，建议将 TiDB 配置项 [`server-version`](/tidb-configuration-file.md#server-version) 设置为 `"5.7.25-TiDB-v7.5.x"`。这是因为，当 TiDB 服务器报告版本为 MySQL 8.0.11 或更高版本时，MySQL Connector/J 会尝试访问 TiDB v7.5.x 中不存在的 `information_schema.KEYWORDS` 表。
+> - MySQL Connector/J 8.0 与 TiDB v7.5.2 或之前版本一起使用时，建议将 TiDB 配置项 [`server-version`](/tidb-configuration-file.md#server-version) 设置为 `"5.7.25-TiDB-v7.5.x"`。这是因为，当 TiDB 服务器报告版本为 MySQL 8.0.11 或更高版本时，MySQL Connector/J 会尝试访问 TiDB v7.5.2 及之前版本中不存在的 [`information_schema.KEYWORDS`](/information-schema/information-schema-keywords.md) 表。
```
Comment (qiancai) [inline develop/dev-guide-choose-driver-or-orm.md:35]:
```suggestion
> - MySQL Connector/J 8.0 与 TiDB v7.5.2 或之前版本一起使用时，建议将 TiDB 配置项 [`server-version`](/tidb-configuration-file.md#server-version) 设置为 `"5.7.25-TiDB-v7.5.x"`。这是因为，当 TiDB 服务器报告版本为 MySQL 8.0.11 或更高版本时，MySQL Connector/J 会尝试访问 TiDB 的 [`information_schema.KEYWORDS`](/information-schema/information-schema-keywords.md) 表，但该表是从 v7.5.3 开始引入的，在 v7.5.2 及之前版本中不存在。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**